lib/LedgerSMB/Database/Change.pm

```diff
@@ -49,6 +49,11 @@ sub new {
     my $self = bless { _path => $path }, $package;
     my @prop_names = qw(no_transactions reload_subsequent);
     $self->{properties} = { map { $_ => $init_properties->{$_} } @prop_names };
+    #This should have been done with '!' in LOADORDER
```

Comment:
Can you be more verbose as to why this can't be solved by adding `'!'` in the LOADORDER file? (or why it's a bad idea to do that)